phpBB/phpbb/passwords/driver/argon2i.php

```diff
@@ -37,21 +37,12 @@ public function __construct(\phpbb\config\config $config, helper $helper, $memor
 	{
 		parent::__construct($config, $helper);
 
-		if ($this->is_sodium())
-		{
-			// For Sodium implementation, set special cost factor values (since PHP 7.4)
-			// See https://wiki.php.net/rfc/sodium.argon.hash and PHPBB3-16266
-			$this->memory_cost = max($memory_cost, 256*1024);
-			$this->threads     = 1;
-			$this->time_cost   = max($time_cost,   3);
-		}
-		else
-		{
-			// Otherwise don't allow cost factors to be below default settings
-			$this->memory_cost = max($memory_cost, 1024);
-			$this->threads     = max($threads,     2);
-			$this->time_cost   = max($time_cost,   2);
-		}
+		// For Sodium implementation, set special cost factor values (since PHP 7.4)
+		// See https://wiki.php.net/rfc/sodium.argon.hash and PHPBB3-16266
+		// Otherwise don't allow cost factors to be below default settings
```

Comment:
Multiline comment best included in a docblock

```
		/**
		 * Set special cost factor values (since PHP 7.4) for Sodium implementation.
		 * @see https://wiki.php.net/rfc/sodium.argon.hash		Documentation.
		 * @see https://tracker.phpbb.com/browse/PHPBB3-16266	Tracker ticket.
		 *
		 * Don't allow cost factors to be below default settings otherwise.
		 */
```

Reply:
It's not a function.

Reply:
`//` should be used for one/two-liners, `/* */` comment blocks otherwise.